src/main/kotlin/com/hibob/academy/service/PetService.kt

```diff
@@ -47,7 +47,7 @@ class PetService @Autowired constructor(
         }
     }
 
-    fun  getPetsByOwnerId(ownerId: Long, companyId: Long): List<PetData> {
+    fun getPetsByOwnerId(ownerId: Long, companyId: Long): List<PetData> {
```

Comment:
In general - you don't have to create a val pets here - you don't use it - only return it - so instead return the `petDao.getPetsByOwnerId(ownerId, companyId)` directly